app/presentation/ImageViewer/ImageViewer.android.tsx

```diff
@@ -94,7 +94,7 @@ function runTiming(clock: any, value: any, dest: any, startStopClock: any = true
 	const config = {
 		toValue: new Value(0),
 		duration: 300,
-		easing: Easing.inOut(Easing.cubic)
+		easing: EasingNode.inOut(EasingNode.cubic)
```

Comment:
https://docs.swmansion.com/react-native-reanimated/docs/2.2.0/migration#2-easing-renamed-to-easingnode